doc/code_architecture.md

```diff
@@ -0,0 +1,39 @@
+No PAB is used here, we use Cardano-Api with custom helpers
+for transaction construction. On-chain scripts are written
+in pure Plutus.
+
+Utilities:
+
+* `HydraAuction.Plutus.Extras` - extension of `PlutusExtra` package,
+   which contains reimpelementation for important `plutus-ledger` stuff
+* `HydraAuction.Plutus.Orphans` - orphans for Plutus stuff.
+* `HydraAuction.Fixture` bunch of known actors (== Cardano keys) to use.
+   Hydra does have one too, but it only have 3 general-use actors.
+
+Library code:
+
+* `HydraAuction.Types` domain types used by scripts.
+   haskell-language-server-wrapperAnd our `Natural` implementation, lol.
+* `HydraAuction.Addresses` - newtypes over our script addresses
+* `HydraAuction.(Tx|Onchain).Common` contains both common domain code
+   and utilities we use (to be fixed),
```

Comment:
What do you mean with `to be fixed`?

Reply:
That we want to separate them by modules in #178